### PR TITLE
Add min password length to the set-password command

### PR DESCRIPTION
**Pull request by aguerra**
Better security and to avoid mistakes.

---
This change is [Reviewable](https://reviewable.io/reviews/luizalabs/teresa-api/248)

### CODE REVIEW

File: pkg/client/password.go [hunk 1]
```diff
@@ -6,12 +6,26 @@ import (
...
+const (
+	minPassLen = 8
+)
+
+func checkPassword(pass string) error {
+	if len([]rune(pass)) < minPassLen {
+		return fmt.Errorf("minimum length is %d", minPassLen)
+	}
+	return nil
+}
```
Comment (drgarcia1986) [inline pkg/client/password.go]: What do you think about writing a test for this function?

File: pkg/client/password.go [hunk 2]
```diff
@@ -6,12 +6,26 @@ import (
 	"github.com/howeyc/gopass"
 )
 
+const (
+	minPassLen = 8
+)
+
+func checkPassword(pass string) error {
+	if len([]rune(pass)) < minPassLen {
+		return fmt.Errorf("minimum length is %d", minPassLen)
```
Comment (drgarcia1986) [inline pkg/client/password.go]: maybe `minimum password length is...`

File: pkg/client/password.go (hunk 1 above)
Comment (aguerra) [inline pkg/client/password.go]: @drgarcia1986 updated. I guess creating specific errors in this case is not worthwhile so I only checked for error and success.

File: pkg/client/password_test.go [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+		pass string
+		ok   bool
+	}{
+		{"12345678", true},
+		{"test", false},
+	}
+
+	for _, tc := range testCases {
+		if err := checkPassword(tc.pass); err != nil && tc.ok {
+			t.Errorf("expected failure, got success")
```
Comment (drgarcia1986) [inline pkg/client/password_test.go]: 💅  I'm bit confused here, please don't get me wrong, right?
What do you think about changing for something like this:
```
if err := checkPassword(tc.in); (err != nil) && tc.expectError {
    t.Errorf("expected error: %v, got error %v", tc.expectError, err)
} 
```
💅

Comment (aguerra) [inline pkg/client/password_test.go]: @drgarcia1986 updated, I think the original logic is wrong, we have to check both cases.

**Pull request merged**